models/purchase_order.py

```diff
@@ -14,6 +14,34 @@ def button_approve(self):
         res = super(PurchaseOrder, self).button_approve()
         return res
 
+    invoice_payment_status = fields.Selection([
+          ('no', 'Nothing to Pay'),
+          ('not_paid', 'Not Paid'),
+          ('in_payment', 'In Payment'),
+          ('paid', 'Paid'),
+          ('partially_paid', 'Partially Paid'),
+      ], string='Invoice Payment Status', compute='_get_invoice_payement_status', store=True, readonly=True)
```

Comment:
@MaganaAsiati 
Here the field is readonly and stored.
But the same field on the sale_order is only in readonly.
Why this difference? 
What's the advantage to store the field and mark it readonly?